scripts/make-bundle

```diff
@@ -30,13 +30,6 @@ BUILDTIME_DEPS = [
     ('wheel', '0.33.6'),
 ]
 PIP_DOWNLOAD_ARGS = '--no-binary :all:'
-# The constraints file is used to lock the version of rsa needed
-# to be 3.4.2 until we can drop py27 support. This lets us ensure
-# we're distributing a copy that works on all supported platforms.
-CONSTRAINTS_FILE = os.path.join(
```

Comment:
I would assume we would have to remove logic wherever this is referenced? It looks like it is used in the `pip download` command? https://github.com/aws/aws-cli/blob/132c9524ba675b08c19c6073e4f3821475f302ed/scripts/make-bundle#L84-L85

Reply:
Yep, good call out. I'm looking at why this wasn't caught in testing but will get it updated.